docs/Changelog.md

```diff
@@ -10648,6 +10648,44 @@ This version of the operator has been available since version 11 of the default
 <dd>tensor of bool, which should be a scalar.</dd>
 </dl>
 
+### <a name="Range-11"></a>**Range-11**</a>
+
+  Generate a tensor containing a sequence of numbers that begin at `start` and extends by increments of `delta` 
+  up to `limit` (exclusive).
+
+#### Version
+
+This version of the operator has been available since version 11 of the default ONNX operator set.
+
+#### Inputs (2 - 3)
+
+<dl>
+<dt><tt>start</tt> : T</dt>
+<dd>Tensor(scalar, or dims=[1]). First entry in the range.</dd>
```

Comment:
I think those inputs should just be scalars as described in your def.cc.

Reply:
Right - I made the change in the def but didn't generate the doc and check-it in. Did that now. #Resolved.